pytest_html/plugin.py

```diff
@@ -276,7 +262,7 @@ def append_log_html(self, report, additional_html):
 
             for section in report.sections:
                 header, content = map(escape, section)
-                log.append(" {0} ".format(header).center(80, "-"))
+                log.append(f" {header} ".center(80, "-"))
```

Comment:
We can replace the `center` with `f-string` formatting.
```suggestion
                log.append(f" {header:-^80} ")
```

as in

```py
>>> f"{'hello':-^15}"                                                                                                                                                                         
'-----hello-----'
```

Reply:
Nice!